geoportailv3/static/js/olcs/toggle3dcomponent.js

```diff
@@ -5,22 +5,26 @@
  *
  * <app-toggle-3d><app-toggle-3d>
  */
-goog.provide('app.toggle3d');
+goog.module('app.olcs.toggle3d');
 
 goog.require('app');
 goog.require('ngeo.olcs.Service');
 
-app.toggle3d = {
+class Controller {
```

Comment:
Any benefits to use a class based controller?

Reply:
No, it is simply more modern.
However you need to:
- have the constructor / class defined at the to level;
- have the `ngInject` annotation.

Otherwise the minified build will not work.